src/deploy/osp_deployer/director.py

```diff
@@ -827,26 +810,17 @@ def setup_networking(self):
                  'sed -i "s|192.168.120.1|' +
                  self.settings.provisioning_gateway + '|" ' + compute_yaml
                  ]
-        if self.settings.is_fx is True:
-            cmds = [
-               'sed -i "s|em1|changeme1|" ' + compute_yaml,
-               'sed -i "s|em3|changeme2|" ' + compute_yaml,
-               'sed -i "s|em2|changeme3|" ' + compute_yaml,
-               'sed -i "s|em4|changeme4|" ' + compute_yaml,
-            ]
-            cmds.extend(cmds_)
-        else:
-            cmds = [
+        cmds = [
                'sed -i "s|em1|changeme1|" ' + compute_yaml,
                'sed -i "s|p3p1|changeme2|" ' + compute_yaml,
                'sed -i "s|em2|changeme3|" ' + compute_yaml,
                'sed -i "s|p3p2|changeme4|" ' + compute_yaml,
                'sed -i "s|em3|changeme5|" ' + compute_yaml,
             ]
```

Comment:
NIT:  indentation of [ ] brackets is inconsistent in the file  Sometime lining up beneath each other other times the ] is lined up with the array name.